xrpl4j-model/src/main/java/org/xrpl/xrpl4j/model/client/nft/NfTokenOfferObject.java

```diff
@@ -0,0 +1,130 @@
+package org.xrpl.xrpl4j.model.client.nft;
```

Comment:
this class should go with the other `LedgerObject`s in `org.xrpl.xrpl4j.model.ledger`